### ISSUE TITLE
Support spo2 and migrate to v2 API

### ISSUE THREAD

**Issue by dvargasfuertes** [dvargas92495/roamjs-oura-ring#3]

**Comment by sonmaximum** [comment]
Hi, I just wanted to let this project's maintainer know that Oura has announced that the V1 version of its API will be discontinued in Jan 2024, and will no longer be available.  

So after that time, this project will need to use the v2 API in order to function.

**Comment by achalaggarwal** [comment]
@dvargas92495 The v1 API has been removed and the extension has stopped working.

**Comment by dvargasfuertes** [comment]
Yup, we are working on the replacement - we need to stand up an api in our backend in order to support
